x-pack/plugins/security_solution/public/management/pages/policy/view/event_filters/empty/policy_event_filters_empty_unassigned.tsx

```diff
@@ -22,15 +21,14 @@ export const PolicyEventFiltersEmptyUnassigned = memo<CommonProps>(({ policyId,
   const { canCreateArtifactsByPolicy } = useUserPrivileges().endpointPrivileges;
   const { onClickHandler, toRouteUrl } = useGetLinkTo(policyId, policyName);
 
-  // TODO: Uncomment this when assign functionality through the flyout is done
-  // const navigateCallback = usePolicyDetailsNavigateCallback();
-  // const onClickPrimaryButtonHandler = useCallback(
-  //   () =>
-  //     navigateCallback({
-  //       show: 'list',
-  //     }),
-  //   [navigateCallback]
-  // );
+  const navigateCallback = usePolicyDetailsNavigateCallback();
```

Comment:
Thanks for uncomment this! I totally forgot it

Reply:
Thanks back to you. 👏 good thing you'd placed these lines already. I'd to uncomment it and it worked out of the box. 😬